integration/BUILD

```diff
@@ -31,3 +31,24 @@ py_test(
         requirement("idna"),
     ],
 )
+
+py_binary(
+    name = "cpp_benchmark_client_server",
+    srcs = [
+        "cpp_benchmark_client_server.py",
+    ],
+    deps = [
```

Comment:
Tiny nit: generally prefer singleton lists to be same line, e.g. `deps = ["foo"],`, if they fit.